model/registry/user.yaml

```diff
@@ -28,8 +28,14 @@ groups:
         type: string
         stability: experimental
         brief: >
-          Unique identifier of the user.
+          Unique identifier of an authenticated user.
         examples: ['S-1-5-21-202424912787-2692429404-2351956786-1000']
+      - id: anonymous_id
+        type: string
+        stability: experimental
+        brief: >
+          Unique identifier of an anonymous user who interacts with a system or service without providing personal information or authentication credentials. It allows the system to track user activity while maintaining anonymity.
```

Comment:
please split the line to fix lint

Reply:
How is `user.anonymous_id` different to the already existing `user.hash` attribute?

From the description both attributes try to achieve the same.

Reply:
`user.hash` is a hash of user.id or user.name. anonymous user id is a completely separate kind of user id, not just anonymizing a well-known user id. For users who have neither a user.id nor a user.name, anonymous user id is still can be tracked via browsers. hope that makes sense.

Reply:
Added this to note.